configs/uhttpConfig.cmake

```diff
@@ -0,0 +1,8 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+include("${CMAKE_CURRENT_LIST_DIR}/uhttpTargets.cmake")
+
+get_target_property(UHTTP_INCLUDES uhttp INTERFACE_INCLUDE_DIRECTORIES)
+
+set(UHTTP_INCLUDES ${UHTTP_INCLUDES} CACHE INTERNAL "")
```

Comment:
Does this need an extra line or do we not care for cmake files?

Reply:
yup newlines everywhere :)